src/main/java/seedu/address/model/role/Properties.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.role;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.property.Property;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a list of properties that a seller is selling / managing.
+ */
+public class Properties {
```

Comment:
I feel like this class could be unnecessary, since we can just represent the properties field in Seller as `Set<Property>`

Reply:
Yep, as discussed in the Telegram group - decided to add Properties as an attribute class in the spirit of AB3